db/schema.rb

```diff
@@ -284,7 +284,6 @@
     t.integer "failed_attempts", default: 0
     t.string "unlock_token"
     t.datetime "locked_at", precision: nil
-    t.string "phone"
```

Comment:
@kiddhustle don't think this should have been here either. As a general rule of thumb should only really have schema changes if you have a migration.